From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jonathan Schwender <schwenderjonathan@gmail.com>
Date: Mon, 27 Jul 2026 20:01:24 +0100
Subject: [PATCH] Silence ptr comparison warning to fix CI

---
 src/lib.rs | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/lib.rs b/src/lib.rs
index 955d0f2..b7ab501 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -1,6 +1,9 @@
 #![allow(non_camel_case_types)]
 #![allow(non_snake_case)]
 #![allow(non_upper_case_globals)]
+// This warning should probably be addressed, but that would be a breaking change.
+// For now we simply silence the warning, to get CI running again.
+#![allow(unpredictable_function_pointer_comparisons)]
 #![deny(missing_copy_implementations)]
 
 use core::ffi::{c_char, c_int, c_long, c_short, c_uchar, c_uint, c_ulong, c_ushort, c_void};
